english/pizza/pizza.md

```diff
@@ -0,0 +1,5 @@
+... reaching over to put the phone down you catch a whiff of your own B.O. and realize you need to become clean. First, you reach over and slap the ass next to you attempting to wake the owner. No reaction, so you get up and go into the shower.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.